tiled/catalog/migrations/versions/a66028395cab_enrich_datasource_asset_association.py

```diff
@@ -0,0 +1,230 @@
+"""Enrich DataSource-Asset association.
+
+Revision ID: a66028395cab
+Revises: 3db11ff95b6c
+Create Date: 2024-01-21 15:17:20.571763
```

Comment:
I have only skimmed through this file, and not looked closely.

Reply:
Very reasonable of you.